notebooks/samples/explanations/tf2/ai-explanations-image.ipynb

```diff
@@ -95,7 +95,7 @@
       "source": [
```

Comment:
<p>Second sentence in "Approximation Error":</p><p>"Otherwise, you can reduce the error by redeploying the model with a higher number of integral steps, or by adjusting the baseline. Learn more about <a href="https://cloud.google.com/ai-platform/prediction/docs/ai-explanations/using-feature-attributions#using-approx-error" target="_blank">using the approximation error</a>."</p><p> </p><p>Last paragraph:</p><p>"Each image is overlaid with feature attributions. Pixels that are highlighted in green contributed positively to the prediction. Areas of the image with denser highlighting are more highly predictive. You can modify the visualization settings for your feature attributions by redeploying the model with your updated settings. Learn more about <a href="https://cloud.google.com/ai-platform/prediction/docs/ai-explanations/visualizing-explanations" target="_blank">visualization settings</a>."</p><p> </p>
<br/>

 _Reply via <a href='https://app.reviewnb.com/GoogleCloudPlatform/ai-platform-samples/pull/206/discussion/'>ReviewNB</a>_ <div id='ReviewNBCommentContext-DoNotDelete' style='display:none' data-state='OPEN' data-cellIndex='81'/>